.github/workflows/branches.yaml

```
@@ -24,7 +24,11 @@ jobs:
       unit_coverage: true
       unconditional: true
       unit_coverage_command: "nix run .#coverage-unit"
+<<<<<<< kauki/ci/publish
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)